### PR TITLE
Added GetAccountsInfo and tests to execute batch of method from foundation

### PR DESCRIPTION
**Pull request by andreevym**

### CODE REVIEW

File: cc/get_accounts_info.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+type GetAccountsInfoResponse struct {
+	Responses []peer.Response
+}
+
+func (c *ACL) GetAccountsInfo(stub shim.ChaincodeStubInterface, _ []string) peer.Response {
+	args := stub.GetArgs()
+	if len(args) == 1 {
+		return shim.Error(fmt.Sprintf("incorrect number of arguments. expecting more 1 but found %d", len(args)))
+	}
```
Comment (pfi79) [inline cc/get_accounts_info.go:28]:
```
for _, a := range args{
........
}
```

File: cc/get_accounts_info.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+	getAccountsInfoRequest := &GetAccountsInfoRequest{}
+	err := json.Unmarshal(args[1], getAccountsInfoRequest)
+	if err != nil {
+		return shim.Error(fmt.Sprintf("failed unmarshal GetAccountsInfoRequest: %s", err))
+	}
```
Comment (pfi79) [inline cc/get_accounts_info.go:34]:
```go
var ss []string
err = json.Unmarshal(a, ss)
```

File: cc/get_accounts_info.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+	responses := make([]peer.Response, 0)
+	for _, item := range getAccountsInfoRequest.Items {
+		var response peer.Response
+		switch item.Method {
+		case "getAccountInfo":
+			for _, address := range item.Args {
+				response = c.GetAccountInfo(stub, []string{address})
+				responses = append(responses, response)
+			}
```
Comment (pfi79) [inline cc/get_accounts_info.go:44]:
```go
switch ss[0] {
		case "getAccountInfo":
                         response = c.GetAccountInfo(stub, ss[1:])
```

File: cc/get_accounts_info.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+		default:
+			responses = append(responses, shim.Error(fmt.Sprintf("failed get accountInfo: unknown method '%s'", item.Method)))
+		}
+	}
+
+	bytes, err := json.Marshal(GetAccountsInfoResponse{Responses: responses})
+	if err != nil {
+		return shim.Error(fmt.Sprintf("failed to marshal GetAccountsInfoResponse: %s", err))
+	}
```
Comment (pfi79) [inline cc/get_accounts_info.go:63]:
```go
proto.Marshal(resposes)
```
?

File: tests/unit/get_accounts_info_test.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	respAddUser := addUser(stub, s)
+	validationResultAddUser(t, stub, respAddUser, s)
+
+	args := make([][]byte, 0)
+	args = append(args, []byte(common.FnGetAccountsInfo))
+	for i := 0; i < 5; i++ {
+		bytes, err := json.Marshal([]string{common.FnGetAccInfoFn, s.testAddress, s.testAddress})
+		require.NoError(t, err)
+		args = append(args, bytes)
+	}
```
Comment (pfi79) [inline tests/unit/get_accounts_info_test.go:37]:
```go
	for i := 0; i < 5; i++ {
		bytes, err := json.Marshal([]string{common.FnGetAccInfoFn, s.testAddress})
		require.NoError(t, err)
		args = append(args, bytes)
	}
```

File: cc/get_accounts_info.go [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+			return shim.Error(fmt.Sprintf("not enough arguments '%s'", string(b)))
+		}
+
+		var response peer.Response
+		switch args[0] {
+		case "getAccountInfo":
+			for _, address := range args[1:] {
+				response = c.GetAccountInfo(stub, []string{address})
+				responses = append(responses, response)
+			}
```
Comment (pfi79) [inline cc/get_accounts_info.go:30]: loops are unnecessary

File: cc/get_accounts_info.go [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+		case "checkAddress":
+			for _, addressBase58Check := range args[1:] {
+				response = c.CheckAddress(stub, []string{addressBase58Check})
+				responses = append(responses, response)
+			}
```
Comment (pfi79) [inline cc/get_accounts_info.go:35]: loops are unnecessary

File: cc/get_accounts_info.go [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+		case "checkKeys":
+			for _, publicKey := range args[1:] {
+				response = c.CheckKeys(stub, []string{publicKey})
+				responses = append(responses, response)
+			}
```
Comment (pfi79) [inline cc/get_accounts_info.go:40]: loops are unnecessary

Comment (andreevym) [reply cc/get_accounts_info.go:40]: if i will remove this loop so we can't make few checks without passing full method name
for example 
with loop:
`[]string{"checkKeys", publicKey1, publicKey2, publicKey3}`

without loop
`[]string{"checkKeys", publicKey1}, []string{"checkKeys", publicKey2}, []string{"checkKeys", publicKey3}`

**Review by pfi79 — APPROVED** [review]

**Pull request merged**